doc/classes/OS.xml

```diff
@@ -132,8 +132,10 @@
 			<return type="Dictionary" />
 			<param index="0" name="path" type="String" />
 			<param index="1" name="arguments" type="PackedStringArray" />
+			<param index="2" name="nonblock" type="bool" default="false" />
```

Comment:
I feel like negative booleans are a bit harder to read, how about calling it `blocking` and defaulting to `true`?

Reply:
Changed it to `blocking`.